src/setupTests.js

```diff
@@ -66,3 +68,49 @@ global.mountWithRouter = children => (path, entries) => {
     )
   };
 };
+
+global.mountWithIntl = children => {
+  return {
+    component: mount(<IntlProvider locale="en">{children}</IntlProvider>)
+  };
+};
+
+global.shallowWithIntl = children => {
+  return {
+    component: shallow(<IntlProvider locale="en">{children}</IntlProvider>)
+  };
+};
+
+global.mountWithIntlProvider = children => initialState => {
+  const store = mockedStore({
+    ...initialState,
+    auth: { ...initialState.auth, locale: 'en' }
+  });
+  return {
+    component: mount(
+      <IntlProvider locale="en" messages={english}>
+        <BrowserRouter keyLength={0}>
+          <Provider store={store}>{children}</Provider>
+        </BrowserRouter>
+      </IntlProvider>
+    ),
+    store
+  };
+};
+
+global.shallowWithIntlProvider = children => initialState => {
```

Comment:
Basically `shallowWithIntlProvider` and `shallowWithProvider` are doing the same thing. We should refactor into  this into `shallowWithProviders` and `mountWithProviders` that sets the redux provider and the intl provider in one function.